website/api/controllers/entrance/signup.js

```diff
@@ -37,6 +37,8 @@ the account verification message.)`,
 
     organization: {
       required: true,
+      // required: false, //« Change organization to be required: false.
+      // defaultsTo: 'N/A', //« Add a default value, using 'N/A' as a default because this is required on the user model.
```

Comment:
I realize this is a change in the requirements, but by convention, I think empty string would make sense here.

FYI @noahtalerman - if there’s a reason not to, let Eric know?  Otherwise let’s go with empty string.

Reply:
The organization is required on the User model, so we can't set it to an empty string.

Reply:
We're about to make it no-longer required though, right?

Reply:
i.e. if you remove `required:true`, the implicit default behavior behind the scenes is `required:false` and `defaultsTo: ''` (which can both be omitted, since they're the default)